README.md

```diff
@@ -29,6 +29,41 @@ Official documentation on how to use this provider can be found on the
 [Terraform
 Registry](https://registry.terraform.io/providers/argoproj-labs/argocd/latest/docs).
 
+## Version Compatibility 
+
+We aim to support `n-2` versions of Argo CD, where `n` is the latest available minor version. Currently this means we support Argo CD `2.14`, `2.13` and `2.12`. 
```

Comment:
Maybe we should simplify this to something like we support current active version of ArgoCD and link to their docs? Explicitly calling out the current version is going to be a maintenance burden?

Reply:
Yeah, exactly. We support all currently active version of ArgoCD.

Reply:
I reworded the section, WDYT now?